src/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+
+const Port = 3000;
```

Comment:
o nome da constante deve ser escrito em minúsculo
```suggestion
const port = 3000;
```